lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -262,7 +262,9 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
         }
 
         if (urlFile?.currentValue) {
-            this.fileName = this.fileName ? this.fileName : this.viewUtilsService.getFilenameFromUrl(urlFile.currentValue);
+            this.fileName = this.fileName
```

Comment:
this can be shorter, just: 

this.fileName ||= this.viewUtilsService.getFilenameFromUrl(urlFile.currentValue);